driver/longhorn

```diff
@@ -74,6 +74,8 @@ dealExistedVolume() {
         echo -ne "Success"
     elif [ "${STATE}" == "detaching" ]; then
         echo -ne "detaching"
+    elif [ "${STATE}" == "" ]; then
```

Comment:
Volume.Status.State can be "" for a brief period of time after volume created and before manager picking it up.

What issue did you get?

Reply:
Ah, yeah you're right...

Every time createVolume is called, logs show `${volumeName} exists but not in a valid state to attach` which was confusing. Instead, when `${OUT}` is empty we can output `${volumeName} doesn't exist`.